src/main/resources/com/google/api/codegen/ruby/version_index.snip

```diff
@@ -36,9 +36,12 @@
 @end
 
 @private readmeModule(indexType, module, iterator, content)
+  @# rubocop:disable LineLength
```

Comment:
I'm pretty sure this will not give the intended results. This will add "rubocop:disable LineLength" at the beginning of the documentation and "rubocop:enable LineLength" at the end of the documentation.